server/src/routes/auth.ts

```diff
@@ -72,33 +69,50 @@ router.post('/register', async (req, res) => {
                     email,
                     name,
                     password,
-                });
-                const userJwt = jwt.sign(
-                    {
-                        name: user.name,
-                        email: user.email,
-                    },
-                    secretKey
-                );
-
-                req.session = {
-                    jwt: userJwt,
-                };
-                req.session.save();
-                res.cookie('Cookie', req.session);
+                })
                 return res.status(200).json({
-                    jwt: userJwt,
-                    user: user,
-                    message: 'Account created successfully',
-                });
+            message: 'created account',
+        });
+          
             }
         }
     } catch (error) {
+        console.log(error)
         console.error('Unable to register');
         return res.status(400).json({
             message: 'Could not create account',
         });
     }
 });
 
+router.get('/login', async function(req, res) {
+    if ((req.session as ISession)._id) {
+        return res.redirect('/');
+    }
+
+    res.setHeader('Content-Type', 'text/HTML');
+    res.write(`
+    <h1>Login</h1>
```

Comment:
We don't need this, since we are not doing server side rendering. So all the login html etc. will be handled on client side. Simply return session information to client side instead